src/gatsby-node.js

```diff
@@ -139,37 +139,60 @@ exports.sourceNodes = async (
   for (let i = 0; i < endpoints.length; i++) {
     const { uid } = endpoints[i];
 
-    await downloadMediaFiles(data[i], ctx, uid);
-
-    for (let entity of data[i]) {
-      const nodes = createNodes(entity, ctx, uid);
-
-      await Promise.all(nodes.map((n) => actions.createNode(n)));
-
-      const nodeType = makeParentNodeName(ctx.schemas, uid);
+    if (uid === 'plugin::upload.file') {
+      for (let file of data[i]) {
```

Comment:
I'm curious if it would be possible to use parallelization for these loops? With Promise.all

Reply:
We are already doing it for all entities of a content type. I will make it parallel for all the entities of the file content type.

Reply:
For the file model it's a bit different I don't want to create children node but just to apply the image processing, I am not even sure we should support it at the moment, it does not really makes sense for query all the files in gatsby.

WDYT about removing it completely?

Reply:
I think the previous plugin didn't have it, and I don't remember anyone in the community asking for it. If it requires special treatment (like no query params etc.) we might as well remove I think

Reply:
Perfect!

It could always be added later if the community needs it anyway